smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -49,6 +51,11 @@ contract MixinKeys is
   // Addresses are never removed by design to avoid abuses around referals
   address[] public owners;
 
+  // A given key has both an owner and a manager.
+  // The owner and manager may be the same address
+  // Each key can have at most 1 keyManager, but may have no manager.
```

Comment:
Could you add a comment to clarify what no manager means?  I thought we were talking about removing that. But the code still supports no manager.

I think I prefer no manager means the keyOwner.  Then we only set it if it's different - e.g. it's 0 for a normal purchase.

Reply:
Looking back over our discussion, we did discuss the merits of both keeping/removing this.
>I think I prefer no manager means the keyOwner. Then we only set it if it's different - e.g. it's 0 for a normal purchase.

 I support this and will modify so that if 0 is passed as the keyManager address, then the  keyOwner is also the manager.